### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [GnuCash/Program](https://hosted.weblate.org/projects/gnucash/gnucash/).

It also includes following components:

* [GnuCash/Glossary](https://hosted.weblate.org/projects/gnucash/glossary/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: po/sv.po [hunk 1]
```diff
@@ -1356,7 +1356,7 @@ msgstr "Symbol"
 #: gnucash/report/trep-engine.scm:925 gnucash/report/trep-engine.scm:1177
 #: gnucash/report/trep-engine.scm:2249
 msgid "Shares"
-msgstr "Andelar"
+msgstr "Aktier"
```
Comment (fellen) [inline po/sv.po:1359]: @bittin At least this change is going in the wrong direction. Did you consider that the term is also used for funds and other commodities?

File: po/sv.po [hunk 2]
```diff
@@ -3451,7 +3451,7 @@ msgstr "Sök order"
 
 #: gnucash/gnome/dialog-payment.c:237
 msgid "You must enter a valid account name for posting."
-msgstr "Du måste ange ett giltigt kontonamn för registrering."
+msgstr "Du måste ange ett giltigt kontonamn för flytt."
```
Comment (fellen) [inline po/sv.po:3454]: @bittin Also this and your following changes seem problematic. They deal with processing invoices, not buy against cash or other direct payment. Perhaps read the [manual](https://gnucash.github.io/gnucash-docs/C/gnucash-manual/chapter_busnss.html) or [guide](https://gnucash.github.io/gnucash-docs/C/gnucash-guide/chapter_bus_features.html)?

**Comment by fellen** [comment]
Thanks!

**Comment by bittin** [comment]
Thanks for reviewing this, and sorry for being tired and making mistakes

**Comment by bittin** [comment]
> Thanks for reviewing this, and sorry for being tired and making mistakes

Arve promised he will fix this so its correct again 
was tired and exhausted during Debcamp going up early Indian time from Sweden my sincere apology for screwing things up

**Pull request merged**